azure-pipelines.yml

```diff
@@ -60,15 +55,21 @@ stages:
         publishTestResults: true
         workingDirectory: 'Fakebook.Profile'
 
-    - script: dotnet publish --configuration $(buildConfiguration) --output publish
-      displayName: dotnet publish
-      workingDirectory: 'Fakebook.Profile/Fakebook.Profile.RestApi'
-
     - task: SonarCloudAnalyze@1
       displayName: sonar run analysis
-
+      
     - task: SonarCloudPublish@1
       displayName: sonar analysis publish
       inputs:
         pollingTimeoutSec: '300'
-      
\ No newline at end of file
+
+- stage: docker
```

Comment:
this stage isnt dependent on the previous one so it can run in parallel - use `dependsOn: []`

Reply:
no, i meant literally `[]`, because it doesn't depend on the previous stage having finished, it can run in parallel